### DISCUSSION TITLE
An error is reported, but I have upgraded Gradio. I still get an error after restarting the operation. How to solve it?

### DISCUSSION THREAD

**Discussion by leozongmac** [lllyasviel/Fooocus#3032] [Q&A] [answered]
IMPORTANT: You are using gradio version 3.41.2, however version 4.29.0 is available, please upgrade.
An error is reported, but I have upgraded Gradio. I still get an error after restarting the operation. How to solve it?
![Uploading Snipaste_2024-05-28_15-32-18.png…]()

**Comment by ILikeToasters** [comment]
That is the version that Fooocus uses and needs. That warning can be ignored.

**Comment by Ckrotky** [comment, marked as answer]
https://github.com/lllyasviel/Fooocus/issues/2911#issuecomment-2108362279
